btf/ext_info.go

```diff
@@ -142,12 +142,15 @@ func AssignMetadataToInstructions(
 
 // MarshalExtInfos encodes function and line info embedded in insns into kernel
 // wire format.
+//
+// If an instruction has an [asm.Comment], it will be synthesized into a mostly
+// empty line info.
 func MarshalExtInfos(insns asm.Instructions, b *Builder) (funcInfos, lineInfos []byte, _ error) {
```

Comment:
There doesn't seem to be a test for this function that I could extend and it also doesn't seem trivial to add one, since I'd have to fake the `Builder` argument. But at least this gets partially tested by the `TestCOREPoisonLineInfo` test.

Reply:
Yea, we don't have a specific test. But pretty much every other test that loads a program with BTF into the kernel invokes this, so we have a lot of indirect coverage. If we break anything in here it will be caught.